MyArrayList/Program.cs

```diff
@@ -14,27 +17,33 @@ static void Main(string[] args)
             list.Add(3);
             list.Add(4);
             list.Add(5);
-
-
         }
     }
 }
 
 public class MyList
 {
-    private int[] _list;
+    // It is better to rename variable _list since it is array and it might be confusing with List<T> class.
+    private int[] _list = new int[0];
+
     //I have created ony the default constructor. Actually, didn't find a reason to add the contructors based on one element or array
     public MyList()
     {
+        // You can save some lines of the code by moving array initialisation to the defenition of array.
+        // Please see my changes. If you move initialisation this constructor can be removed.
         _list = new int[0];
     }
 
-    // Adds element to the end
+    /// <summary>
+    /// Adds element to the end
+    /// </summary>
+    /// <param name="num"></param>
     public void Add(int num)
     {
         Array.Resize(ref _list, _list.Length + 1);
         _list[_list.Length - 1] = num;
     }
+    
     //Deletes the first element equal to the value
```

Comment:
Actualy it deletes ALL elements with given value. And seems like this code wont be working correctly. 
Please debug using array = {1, 1, 1, 2, 3, 3, 4 } and try to remove '1' or '3'.